### PR TITLE
[TS-166] Locker package usage example

### PR DESCRIPTION
**Pull request by im-kulikov**

### CODE REVIEW

File: examples/locker/main.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		locker.Logger(nop.New()),
+		locker.Redis(store),
+	)
+
+	wg := sync.WaitGroup{}
+
+	for i := 0; i <= 10; i++ {
+		wg.Add(1)
+		go func(index int) {
+			lock.Run("my-key", time.Second*10, func() error {
```
Comment (jenchik) [inline examples/locker/main.go:30]: не проверяется что мы действительно не можем параллельно выполнить работу

Comment (im-kulikov) [reply examples/locker/main.go:30]: what're you talk about?

Comment (jenchik) [reply examples/locker/main.go:30]: example:
goroutine #1 do something
and goroutine #2 do something in this time

Comment (jenchik) [reply examples/locker/main.go:30]: or not necessary

Comment (im-kulikov) [reply examples/locker/main.go:30]: nope

they are unable to do something at the same time,
because we use redis lock (bsm/redis-lock)

Comment (im-kulikov) [reply examples/locker/main.go:30]: I showed how to use the package, but not describe it

to see how it work, you can look at source code

**Review by pcherednichenko — APPROVED** [review]

**Review by kpeo — APPROVED** [review]

File: examples/locker/main.go (hunk 1 above)
Comment (jenchik) [reply examples/locker/main.go:30]: ok

**Review by jenchik — APPROVED** [review]

**Pull request merged**